python/tvm/relay/analysis/analysis.py

```diff
@@ -384,6 +384,22 @@ def extract_fused_functions(mod):
     return ret
 
 
+def list_op_frequencies(mod):
```

Comment:
nit: Personally prefer (but not enforce) a short name.

```suggestion
def list_op_freqs(mod):
```